runner/src/main/java/org/ananas/runner/steprunner/jdbc/pgsql/PostgresqlDataTypes.java

```diff
@@ -36,6 +44,10 @@
       "timestamp with time zone",
       Schema.FieldType.DATETIME.withMetadata("subtype", "TS_WITH_LOCAL_TZ").withNullable(true),
       true),
+  TIME_WITH_TIME_ZONE_metadata(
+      "time with time zone",
+      Schema.FieldType.DATETIME.withMetadata("subtype", "TS_WITH_LOCAL_TZ").withNullable(true),
+      true),
```

Comment:
Need to review these mappings here. Scheme.FieldType -> data type literal mapping seems not working. Pay attention to the default value. Some of them override others